templates/vue/src/components/Lightbox/index.vue

```diff
@@ -292,6 +298,9 @@ export default {
         height: this.lightboxDimensions.height,
       }
     },
+    handleTabChange(newTab) {
+      this.selectedTab = newTab
```

Comment:
Since `selectedTab` is not used anywhere here, this whole method and the event handler on the navbar component can be removed too?

Reply:
Ditto as above